CRM/Import/Forms.php

```diff
@@ -108,7 +108,6 @@ protected function getUserJobSubmittedValues(): array {
     // other imports, ditto uploadFile.
     'skipColumnHeader' => 'DataSource',
     'fieldSeparator' => 'DataSource',
-    'uploadFile' => 'DataSource',
```

Comment:
Removing it from here means it is no longer saved beyond the life of the `DataSource` form - ie to the `UserJob` table